### PR TITLE
Feature: Restyling Login Form

### PR DESCRIPTION
**Pull request by waylonturbes**
Co-authored-by: @PriscilaMonteiro @ModupeD

## Description

- Renamed `signin.less` to `login.less`
- Refactored `login.less` and added new styles (login page is now fully responsive for smaller screens)
- Changed Okta sign-in widget in `LoginContainer.js` so that it displays a register link that routes the user to the register page
Note: register page has not been built yet
- Added new styles to `themeOverrides.js` and changed some

## Video

https://youtu.be/YY6OEWyDRS4

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by mcmillanj — APPROVED** [review]
Good work

**Review by manny-glez — APPROVED** [review]
Good work, I saw this during product review, it's looking way much better with these changes.

**Review by ElchinNasirov — APPROVED** [review]
Great work.

File: src/styles/signin.less [hunk 1]
```diff
@@ -1,41 +1,79 @@
...
+    }
+
+    &:focus {
+      background: #263d47;
+      border-radius: 8px;
+    }
+  }
 
-#okta-sign-in.auth-container .okta-sign-in-header{
+  .okta-sign-in-header {
```
[the remark this answers is not in the document]
Comment (PriscilaMonteiro) [reply src/styles/signin.less]: Solved.

File: src/styles/login.less [hunk 2]
```diff
@@ -0,0 +1,126 @@
+.input-focus() {
+  border: @input-focused-border;
```
Comment (krisf451) [inline src/styles/login.less:2]: Love the use of global less variables, keeps it clean.

**Review by krisf451 — APPROVED** [review]

**Comment by ModupeD** [comment]
This looks amazing

**Pull request merged**